pom.xml

```diff
@@ -36,8 +36,12 @@
     <gson.version>2.8.2</gson.version>
     <jackson.version>2.5.2</jackson.version>
     <jmockit.version>1.41</jmockit.version>
+    <mockito.version>2.24.0</mockito.version>
     <junit.version>4.12</junit.version>
     <okhttp3.version>4.9.1</okhttp3.version>
+    <guava.version>27.0.1-jre</guava.version>
+    <gcs.client.version>1.62.0</gcs.client.version>
```

Comment:
Any specific reason to use an older gcs client version?
For java 8, we should be able to use `com.google.cloud:google-cloud-storage:2.6.0`.
This becomes more important as client versions below `2.3.0` cannot write to bucket with new lifecycle actions.

Reply:
No, No specific reason, it's just old habit of using this version.
I have upgraded the version will push the in the next PR if there's no other rework required else will push along with rework.

Reply:
Fixed